.oca/oca-port/blacklist/mail_tracking_mass_mailing.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pull_requests": {
+    "1244": "skip precommit changes",
+    "1048": "(auto) Nothing to port from PR #1048"
```

Comment:
This will probably not work as expected here, as this is a new repo.
These PR ids are from OCA/social

@sebalix

Reply:
Hi, indeed, it should be dropped

Reply:
Yes, there is a fix regarding that has been merged lately to store the full PR ref: https://github.com/OCA/oca-port/pull/57